hathor/transaction/static_metadata.py

```diff
@@ -181,7 +174,6 @@ def _get_previous_feature_activation_bit_counts(
         return parent_block.static_metadata.feature_activation_bit_counts
 
 
-@dataclass(slots=True, frozen=True, kw_only=True)
```

Comment:
Why are you removing this line?

Reply:
Please see https://github.com/HathorNetwork/hathor-core/pull/1016#discussion_r1729346188